migrations/20210730153858_fill_university_address_fields.ts

```diff
@@ -0,0 +1,33 @@
+import { Knex } from 'knex';
```

Comment:
Not sure doing a migration for this is a good idea.. i would prefer a one time script

Reply:
Ok I'll update it.

- [ ] Fill addresses with one time script instead of knex migration